src/hearings/containers/hearing-actuals/hearing-actual-add-edit-summary/hearing-actual-add-edit-summary.component.html

```diff
@@ -49,6 +49,7 @@ <h1 class="govuk-heading-m">Hearing stage and result</h1>
       </dl>
     </div>
     <br>
+    <h1 class="govuk-heading-m">{{ actualHearingDays && actualHearingDays.length > 1 ? 'Hearing date(s)' : 'Hearing date' }}</h1>
```

Comment:
seems duplicated with line 25, can these two be extracted to ts and call the same method to get the display value.